src/test/java/newTest/LogInTestNew.java

```diff
@@ -9,6 +9,8 @@ public class LogInTestNew extends Baseclass {
     public void start() throws InterruptedException
```

Comment:
What is Start can be rename the class to more descriptive so that we can identify the test